aa_parametersets/aap_parameters_defaults.xml

```diff
@@ -109,6 +109,8 @@
         <email desc='Send an email when analysis is done or breaks?' ui='text'></email>
         <NIFTI4D desc='Use NIFTI-4D instead of NIFTI-3D' ui='yesno'>1</NIFTI4D>
         <hardlinks desc='Allow hardlinks during retrieving inputs' ui='yesno'>1</hardlinks>
+        <remotesymlinks desc='Allow symlinks when doing remote pipeline connection when receiving inputs?' ui='yesno'>0</remotesymlinks>
+        <symlinks desc='Allow symlinks during retrieving inputs' ui='yesno'>0</symlinks>
```

Comment:
If `aap.options.symlinks` corresponds to symlinks within a pipeline (i.e., without connection), then please describe accordingly. E.g.: "Allow symlinks during retrieving **local** inputs".